scripts/mktaglist.py

```diff
@@ -48,6 +48,7 @@ def DownloadMasterlist(repository, destinationPath):
     (u'Oblivion', 'Oblivion.esm', 'oblivion', loot_api.GameType.tes4),
     (u'Skyrim', 'Skyrim.esm', 'skyrim', loot_api.GameType.tes5),
     (u'Skyrim Special Edition', 'Skyrim.esm', 'skyrimse', loot_api.GameType.tes5se),
+    (u'Skyrim VR', 'Skyrim.esm', 'skyrimse', loot_api.GameType.tes5se),
```

Comment:
There is no LOOT for skyrim VR yet so leaving this set to tes5se.

Reply:
The latest loot API includes tes5vr now.